diversity-lunch-be/src/test/java/de/brockhausag/diversitylunchspringboot/meeting/controller/MeetingControllerTest.java

```diff
@@ -101,6 +101,32 @@ void testDeleteMeeting_serviceDeleteMeeting() {
         assertEquals(HttpStatus.OK, response.getStatusCode());
     }
 
+    @Test
+    void testDeleteMeeting_isNotDeleted_If_meetingProposalIsMatched() {
+        long id = 1L;
+        MeetingProposalEntity meetingProposalEntity1 = meetingTestdataFactory.entity();
+        meetingProposalEntity1.setMatched(true);
```

Comment:
Ist das setzen des Matched Attribut nicht überflüssig, wenn das in  Zeile 110 sowieso gemocked wird ?